src/Components/Workspace/Transcripts/index.js

```diff
@@ -23,6 +23,19 @@ const Transcripts = ({ projectId, firebase }) => {
     return `/projects/${ projectId }/transcripts/${ id }/correct`;
   };
 
+  const usePreventWindowUnload = (uploading) => {
+    useEffect(() => {
+      if (!uploading) return;
+      window.onbeforeunload = () => {
+        return '';
+      };
+    }, [ uploading ]);
+  };
+  const CheckIfUploading = () => {
+    const stillUploading = items.find(item => item.status == 'uploading');
+    usePreventWindowUnload(stillUploading);
```

Comment:
It looks like this is going to call `usePreventWindowUnload` whether or not any item is returned in the find method. I think this would need to look something like:

`stillUploading ? usePreventWindowUnload : return;`

So only call `prevent` if you've got something in progress